2019-11-27 22:28:48,303 DEBUG   MainThread:51031 [wandb_config.py:_load_defaults():110] no defaults not found in config-defaults.yaml
2019-11-27 22:28:48,309 DEBUG   MainThread:51031 [cmd.py:execute():722] Popen(['git', 'cat-file', '--batch-check'], cwd=/Users/jeongsooha/project/ReinforcementLearning, universal_newlines=False, shell=None, istream=<valid stream>)
2019-11-27 22:28:48,320 DEBUG   MainThread:51031 [cmd.py:execute():722] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=/Users/jeongsooha/project/ReinforcementLearning, universal_newlines=False, shell=None, istream=None)
2019-11-27 22:28:48,330 DEBUG   MainThread:51031 [cmd.py:execute():722] Popen(['git', 'status', '--porcelain', '--untracked-files'], cwd=/Users/jeongsooha/project/ReinforcementLearning, universal_newlines=False, shell=None, istream=None)
2019-11-27 22:28:48,349 DEBUG   MainThread:51031 [run_manager.py:__init__():535] Initialized sync for reinforce/imva38xe
2019-11-27 22:28:48,351 INFO    MainThread:51031 [run_manager.py:wrap_existing_process():1131] wrapping existing process 51024
2019-11-27 22:28:48,355 DEBUG   MainThread:51031 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): pypi.org:443
2019-11-27 22:28:48,659 DEBUG   MainThread:51031 [connectionpool.py:_make_request():437] https://pypi.org:443 "GET /pypi/wandb/json HTTP/1.1" 200 37940
2019-11-27 22:28:48,736 INFO    MainThread:51031 [run_manager.py:init_run():916] system metrics and metadata threads started
2019-11-27 22:28:48,737 INFO    MainThread:51031 [run_manager.py:init_run():950] upserting run before process can begin, waiting at most 10 seconds
2019-11-27 22:28:48,745 DEBUG   Thread-14 :51031 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): api.wandb.ai:443
2019-11-27 22:28:49,178 DEBUG   Thread-14 :51031 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 555
2019-11-27 22:28:49,184 INFO    Thread-14 :51031 [run_manager.py:_upsert_run():1035] saving patches
2019-11-27 22:28:49,185 DEBUG   Thread-14 :51031 [cmd.py:execute():722] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=/Users/jeongsooha/project/ReinforcementLearning, universal_newlines=False, shell=None, istream=None)
2019-11-27 22:28:49,196 DEBUG   Thread-14 :51031 [cmd.py:execute():722] Popen(['git', 'diff', '--cached', '--abbrev=40', '--full-index', '--raw'], cwd=/Users/jeongsooha/project/ReinforcementLearning, universal_newlines=False, shell=None, istream=None)
2019-11-27 22:28:49,208 DEBUG   Thread-14 :51031 [cmd.py:execute():722] Popen(['git', 'version'], cwd=/Users/jeongsooha/project/ReinforcementLearning, universal_newlines=False, shell=None, istream=None)
2019-11-27 22:28:49,240 DEBUG   Thread-14 :51031 [cmd.py:execute():722] Popen(['git', 'merge-base', 'HEAD', 'a566c4c2d6adb1bfc9e9e524e1d3ed54c5a60c75'], cwd=/Users/jeongsooha/project/ReinforcementLearning, universal_newlines=False, shell=None, istream=None)
2019-11-27 22:28:49,272 INFO    Thread-14 :51031 [run_manager.py:_upsert_run():1039] saving pip packages
2019-11-27 22:28:49,273 INFO    Thread-14 :51031 [run_manager.py:_upsert_run():1041] initializing streaming files api
2019-11-27 22:28:49,274 INFO    Thread-14 :51031 [run_manager.py:_upsert_run():1048] unblocking file change observer, beginning sync with W&B servers
2019-11-27 22:28:49,274 INFO    MainThread:51031 [run_manager.py:wrap_existing_process():1148] informing user process we are ready to proceed
2019-11-27 22:28:49,277 INFO    MainThread:51031 [run_manager.py:_sync_etc():1255] entering loop for messages from user process
2019-11-27 22:28:49,279 DEBUG   Thread-15 :51031 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): api.wandb.ai:443
2019-11-27 22:28:49,313 INFO    Thread-3  :51031 [run_manager.py:_on_file_modified():682] file/dir modified: /Users/jeongsooha/project/ReinforcementLearning/a3c/wandb/run-20191127_132847-imva38xe/config.yaml
2019-11-27 22:28:49,321 DEBUG   Thread-3  :51031 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): api.wandb.ai:443
2019-11-27 22:28:49,584 INFO    MainThread:51031 [run_manager.py:_sync_etc():1278] received message from user process: {"exitcode": 0}
2019-11-27 22:28:49,584 INFO    MainThread:51031 [run_manager.py:_sync_etc():1364] closing log streams and sending exitcode to W&B
2019-11-27 22:28:49,584 INFO    MainThread:51031 [run_manager.py:shutdown():1055] shutting down system stats and metadata service
2019-11-27 22:28:49,652 DEBUG   Thread-15 :51031 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 46
2019-11-27 22:28:49,726 DEBUG   Thread-3  :51031 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 603
2019-11-27 22:28:49,731 INFO    Thread-3  :51031 [run_manager.py:_on_file_created():671] file/dir created: /Users/jeongsooha/project/ReinforcementLearning/a3c/wandb/run-20191127_132847-imva38xe/upstream_diff_a566c4c2d6adb1bfc9e9e524e1d3ed54c5a60c75.patch
2019-11-27 22:28:49,731 INFO    Thread-3  :51031 [run_manager.py:_on_file_created():671] file/dir created: /Users/jeongsooha/project/ReinforcementLearning/a3c/wandb/run-20191127_132847-imva38xe/output.log
2019-11-27 22:28:49,731 INFO    Thread-3  :51031 [run_manager.py:_on_file_created():671] file/dir created: /Users/jeongsooha/project/ReinforcementLearning/a3c/wandb/run-20191127_132847-imva38xe/wandb-metadata.json
2019-11-27 22:28:49,732 INFO    Thread-3  :51031 [run_manager.py:_on_file_created():671] file/dir created: /Users/jeongsooha/project/ReinforcementLearning/a3c/wandb/run-20191127_132847-imva38xe/requirements.txt
2019-11-27 22:28:49,732 INFO    Thread-3  :51031 [run_manager.py:_on_file_created():671] file/dir created: /Users/jeongsooha/project/ReinforcementLearning/a3c/wandb/run-20191127_132847-imva38xe/diff.patch
2019-11-27 22:28:49,740 INFO    MainThread:51031 [run_manager.py:shutdown():1067] stopping streaming files and file change observer
2019-11-27 22:28:50,312 INFO    Thread-3  :51031 [run_manager.py:_on_file_modified():682] file/dir modified: /Users/jeongsooha/project/ReinforcementLearning/a3c/wandb/run-20191127_132847-imva38xe/wandb-metadata.json
2019-11-27 22:28:50,313 INFO    MainThread:51031 [run_manager.py:_on_file_created():671] file/dir created: /Users/jeongsooha/project/ReinforcementLearning/a3c/wandb/run-20191127_132847-imva38xe/wandb-events.jsonl
2019-11-27 22:28:50,321 DEBUG   MainThread:51031 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): api.wandb.ai:443
2019-11-27 22:28:50,664 DEBUG   MainThread:51031 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 603
2019-11-27 22:28:51,315 DEBUG   Thread-18 :51031 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): api.wandb.ai:443
2019-11-27 22:28:51,325 DEBUG   Thread-7  :51031 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): api.wandb.ai:443
2019-11-27 22:28:51,629 DEBUG   Thread-18 :51031 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 811
2019-11-27 22:28:51,641 DEBUG   Thread-18 :51031 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): storage.googleapis.com:443
2019-11-27 22:28:51,649 DEBUG   Thread-7  :51031 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /files/jeongsoo-ha1210/reinforce/imva38xe/file_stream HTTP/1.1" 200 311
2019-11-27 22:28:51,835 DEBUG   Thread-7  :51031 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /files/jeongsoo-ha1210/reinforce/imva38xe/file_stream HTTP/1.1" 200 308
2019-11-27 22:28:51,836 INFO    MainThread:51031 [run_manager.py:_sync_etc():1416] syncing files to cloud storage
2019-11-27 22:28:52,106 DEBUG   Thread-18 :51031 [connectionpool.py:_make_request():437] https://storage.googleapis.com:443 "PUT /wandb-production.appspot.com/jeongsoo-ha1210/reinforce/imva38xe/___batch_archive_1.tgz?Expires=1574861391&GoogleAccessId=gorilla-cloud-storage%40wandb-production.iam.gserviceaccount.com&Signature=X6hZmTKve%2FyON0P7d53658Eawsw34AObtSTDhGG6gbuVzSsKDJFt6L7smFdpvYWFWiC%2FopAoxin8OYUaDy5VWNBsuRmFstj3uaitgvI3GFBED8%2FzwESUj2YKrB7dTttW6Ie%2FsA%2FpJth%2Bys%2FRGmPha%2BSCar%2FgeRwwCOEvsRG%2FNEj5t9WUkOUm%2Fs6J2M2wCc1t1E2%2F06zFGvKtO%2FE4IWF%2F%2Ffdnvgs3L75y8DNpEQ34h%2FPZmmtWfSqYyDzT8dtYahoBM6%2FQ5dNETDVxR5XzKi4AubwNcSMh7dElCmKZtkVNQU%2BXIavuyD0%2FMUPw%2BOCdN0vMmTVdiA6ie%2FkbRt9QGymclw%3D%3D HTTP/1.1" 200 0
2019-11-27 22:28:53,357 INFO    MainThread:51031 [run_manager.py:_sync_etc():1431] syncing complete: https://app.wandb.ai/jeongsoo-ha1210/reinforce/runs/imva38xe
2019-11-27 22:28:53,357 INFO    MainThread:51031 [imva38xe:run_manager.py:_sync_etc():1431] syncing complete: https://app.wandb.ai/jeongsoo-ha1210/reinforce/runs/imva38xe
